Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6181,6 +6182,32 @@ def com_google_fonts_check_colorfont_tables(ttFont):
     else:
         yield PASS, "Looks good!"
 
+@check(
+    id = "com.google.fonts/check/space_on_gid1_for_colrv0",
+    rationale = """
+        A shaping bug in Windows 10 inserts whichever glyph is on GID 1
+        on top of color glyphs, spaces, and also seemingly at random,
+        for COLRv0 fonts.
+
+        Having the space glyph on GID 1 is a practical workaround for that.
+
+        See https://github.com/googlefonts/gftools/issues/609
+    """,
+    proposal = 'https://github.com/googlefonts/fontbakery/issues/3886'
+)
+def com_google_fonts_check_space_on_gid1_for_colrv0(ttFont):
+    """Put space glyph on GID 1 right after the .notdef glyph for COLRv0 fonts."""
+    SUGGESTED_FIX = ("To fix this, please reorder the glyph order so that"
+                     " the space glyph is on GID 1 right after the .notdef glyph.")
+    if 'COLR' in ttFont.keys() and ttFont['glyf'].glyphOrder[1] != 'space':
```

Comment:
don't rely on the name being 'space', even the presence of a glyph that's mapped to SPACE character 0x0020 is not guaranteed nor a requirement as far as I know.